### PR TITLE
area/ui: Add label-value autocompletion

### LINKED ISSUE #348: Autocompletion in parca UI
Will be nice to have autocompletion in parca ui, at least for namespaces in pods

### PR DESCRIPTION
**Pull request by yomete**
Resolves #348 #473 

The PR adds the label value autocompletion to the Parca UI. Parca users will now get a complete suggestion when trying to filter profiles. It also include little UI updates like label-value being displayed as tags after a complete suggestion.

image

### CODE REVIEW

File: ui/packages/app/web/src/components/ui/Card.tsx [hunk 1]
```diff
@@ -21,10 +21,12 @@ const Card = ({children, className = ''}: CardProps) => {
   );
 };
 
-const Header = ({children}: {children: React.ReactNode}) => {
+const Header = ({children, className}: {children: React.ReactNode; className?: string}) => {
```
Comment (manojVivek) [inline ui/packages/app/web/src/components/ui/Card.tsx:24]: Should we also assign a default parameter value for `className` to be `''` as it is an optional, and we have a `className.length` below?

File: ui/packages/shared/components/src/MatchersInput/index.tsx [hunk 2]
```diff
@@ -166,31 +263,85 @@ const MatchersInput = ({
...
+    let suggestion = getSuggestion(suggestionIndex);
+
+    if (suggestion.type === 'labelValue') {
+      suggestion.value = addQuoteMarks(suggestion.value);
+    }
+
     const newValue = complete(suggestion);
     resetHighlight();
+
+    if (suggestion.type === 'labelName') {
```
Comment (manojVivek) [inline ui/packages/shared/components/src/MatchersInput/index.tsx:275]: Shall we make these literals as constants? That will help us define even stricter type for this field.

File: parca.yaml [hunk 3]
```diff
@@ -13,4 +13,4 @@ scrape_configs:
     scrape_interval: "3s"
     scrape_timeout: "2s"
     static_configs:
-      - targets: [ '127.0.0.1:7070' ]
+      - targets: [ "127.0.0.1:7070" ]
```
Comment (metalmatze) [inline parca.yaml:16]: This change seems unrelated.
I'd leave it as is, probably different editors will do different things and there will always be a back and forward. :grimacing:

**Review by metalmatze — COMMENTED** [review]
This will be so good! Looking forward :star_struck:

**Comment by brancz** [comment]
This is really awesome, nice work!
